fiftyone/core/collections.py

```diff
@@ -10415,6 +10415,7 @@ def _build_batch_pipeline(self, aggs_map):
                     "Batchable aggregations must have pipelines with a single "
                     "$project stage; found %s" % _pipeline
                 )
+        project["_id"] = False
```

Comment:
I agree with this change!

For completeness, we should also add this idea to `Values.to_mongo()` class so that we also omit `_id` from any `Values()` aggregations that are "big" (`agg._has_big_result == True`) but not "batchable" (`agg._is_big_batchable == False`)

Reply:
Also, since we're engaging in optimizations like this, we should go ahead and optimize `values("id")` as proposed in https://github.com/voxel51/fiftyone/pull/5479, because that's the most common `values()` operation!

Reply:
Ok I was 100% sure about big batchable so I started there but I didn't look into the other cases. We can do that.

And yeah let's do it, I'll get to that one ASAP

Reply:
@swheaton I already had most of the desired things in https://github.com/voxel51/fiftyone/pull/5479, so I went ahead and added this idea to that PR. It's ready for review now

Reply:
Ok. Still testing stuff.